app/controllers/formbuilder/edit/code.js

```diff
@@ -83,4 +98,49 @@ export default class FormbuilderEditCodeController extends Controller {
       return;
     }
   }
+
+  getConsoleSeverity(severity) {
+    const baseClass = 'code-edit-message ';
+    const mapping = {
+      [SHACL_SEVERITY_TYPE.error]: {
+        type: SHACL_SEVERITY_TYPE.error,
+        icon: 'cross',
+        class: baseClass + 'code-edit-message--bg-error',
+        iconClass: 'code-edit-message--icon-error',
+      },
+      [SHACL_SEVERITY_TYPE.warning]: {
+        type: SHACL_SEVERITY_TYPE.warning,
+        icon: 'alert-triangle',
+        class: baseClass + 'code-edit-message--bg-warning',
+        iconClass: 'code-edit-message--icon-warning',
+      },
+      [SHACL_SEVERITY_TYPE.info]: {
+        type: SHACL_SEVERITY_TYPE.info,
+        icon: 'info-circle',
+        class: baseClass + 'code-edit-message--bg-info',
+        iconClass: 'code-edit-message--icon-info',
+      },
```

Comment:
Works with the values got from the SHACL report. Type stylings and icons are done like this as it was the only way I could think of doing this